app/src/main/java/com/team3663/scouting_app/utility/Logger.java

```diff
@@ -56,74 +53,58 @@ public void clear() {
 
     // Member Function: Close out the logger.  Write out all of the non-time based match data and close the files.
     public void close() {
-        String path = appContext.getString(R.string.logger_path);
-        boolean rc = true;
-
         // If this is a practice, there's nothing to do
         if (Globals.isPractice) return;
 
-        // Ensure the path (if it's not blank) has a trailing delimiter
-        if (!path.isEmpty()) {
-            if (!path.endsWith("/")) path = path + "/";
-        }
-
         // Define the filenames/files to be used for this logger
-        String filename_data = path + Globals.CurrentCompetitionId + "_" + Globals.CurrentMatchNumber + "_" + Globals.CurrentDeviceId + "_d.csv";
-        String filename_event = path + Globals.CurrentCompetitionId + "_" + Globals.CurrentMatchNumber + "_" + Globals.CurrentDeviceId + "_e.csv";
-
-        File file_data = new File(Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOCUMENTS), filename_data);
-        File file_event = new File(Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOCUMENTS), filename_event);
-
-        // Ensure the directory structure exists first - only need to check with one file
-        if (!Objects.requireNonNull(file_data.getParentFile()).exists()) {
-            rc = Objects.requireNonNull(file_data.getParentFile()).mkdirs();
-            if (!rc) Toast.makeText(appContext, "Failed to create directory: " + file_data.getParentFile().getName(), Toast.LENGTH_LONG).show();
-        }
+        String filename_data = Globals.CurrentCompetitionId + "_" + Globals.CurrentMatchNumber + "_" + Globals.CurrentDeviceId + "_d.csv";
+        String filename_event = Globals.CurrentCompetitionId + "_" + Globals.CurrentMatchNumber + "_" + Globals.CurrentDeviceId + "_e.csv";
```

Comment:
Not super important, but this feels very inefficient cause they are the same except for the `_d` and `_e`.

Reply:
any suggestions?

Reply:
Have one variable that is everything but the `_d.csv` or `_e.csv` then just add that in when you use it.

Reply:
Sure, but then when I use it, there's more calculations.  I DO like this since HOW we name it is contained to these lines (in case we change it later).

Reply:
I think it is fine. It just bothered my slightly, seeing it.